.github/workflows/ci.yml

```diff
@@ -1,6 +1,10 @@
 name: CI
 on:
   pull_request:
+    branches:
+      # Branches that modify spack.yaml must be of the form: pre-*.*.* (ex: pre-2024.01.1)
+      # in order for the PR to access the '* Prerelease' GitHub Environments.
+      - 'pre-*.*.*'
```

Comment:
I can't comment on it directly because it hasn't changed, but the `dorny/paths-filter` action should use a commit hash as per our new guidelines/standards.

Reply:
Good point!

Reply:
Done :)